docs/development.md

```diff
@@ -62,27 +62,7 @@ If you started the server in the background, run the following command to stop i
 
 If the server is running in the foreground, press `Ctrl + C` to stop it or run the above command in a new console.
 
-### With Eclipse
```

Comment:
By moving only the IDE workflow to the IDE document, this document becomes the command-line only work workflow but it is still partially required when you work in an IDE workflow. You have left this in an inconsistent state.